recirq/qml_lfe/learn_dynamics_q.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Draw two copy data from 1D random scrambling or tsym style circuits.
+
+learn_dynamics_q.py --n=6 --depth=5 --n_data=20 --batch_size=5 --n_shots=1000
+
+Will create 20 scrambling and 20 tsym circuits (40 total circuits) all on
+12 qubits with depth 5. Once the circuits are generated, batch_size circuits
+will be sent for simulation/execution, drawing n_shots samples from each
+one using `run_batch` in Cirq. By default the bitstring data will be saved
+in the data folder. One can also set `use_engine` to True in order to
+run this against a processor on quantum engine.
+"""
+
+from typing import List
+
+import os
+import cirq
+import numpy as np
+from . import circuit_blocks
+from . import dynamics_flags
+from . import util
+from absl import app
+from absl import logging
+
+FLAGS = dynamics_flags.FLAGS
+
+
+def _build_circuit(
+    qubit_pairs: List[List[cirq.Qid]], use_tsym: bool, depth: int
+) -> cirq.Circuit:
+    """Generate twocopy 1D tsym or scrambling circuit with given depth on qubit_pairs."""
+    inter_gen = circuit_blocks.scrambling_block
```

Comment:
using an `else` clause might get your intention across clearer

Reply:
I'm personally a fan of not having the `else` so I left it as is.